conference.js

```diff
@@ -1827,5 +1773,37 @@ export default {
         if(room) {
             room.sendApplicationLog(JSON.stringify({name, value}));
         }
+    },
+    /**
+     * Disconnect from the conference and optionally request user feedback.
+     * @param {boolean} [requestFeedback=false] if user feedback should be
+     * requested
+     */
+    hangup (requestFeedback = false) {
+        const errCallback = (err) => {
```

Comment:
This is not new code. I moved the hangup function to the conference object. But I'll fix it.